tests/qchem/of_tests/test_convert.py

```diff
@@ -942,6 +942,68 @@ def test_import_state(molecule, basis, symm, method, wf_ref):
     assert np.allclose(wf_comp, wf_ref) or np.allclose(wf_comp, -wf_ref)
 
 
+@pytest.mark.parametrize(
+    ("detscoeffs", "wf_ref"),
+    [
+        (
+            # dmrg
+            (np.array([[0, 3], [3, 0]]).numpy(), np.array([-0.10660077, 0.9943019]).numpy()),
```

Comment:
It should be a list not an array:` [[0, 3], [3, 0]]`

Reply:
Done.